test.js

```diff
@@ -0,0 +1,16 @@
+function main(a, b) {
+  switch (cond) {
+  case "one":
+    doSomething(); // JSHint will warn about missing 'break' here.
+  case "three":
+    doSomethingElse();
+  case "two":
```

Comment:
Commenting in the "files changed" tab instead of individual commits makes for nicer messages in the Conversation tab.

I have seen people tako commens on commits as regural commens on the PR and not something that refers to a line of code.